files/templates/additional-scrape-configs.template.yaml

```diff
@@ -259,7 +257,7 @@
   - target_label: app
     replacement: etcd
   - source_labels: ['__address__']
-    target_label:  'instance'
+    target_label: 'instance'
```

Comment:
why is there quotes here ?

Reply:
We have some for role: nodes so we need to change the port and managed apps as well
Also, the kubernetes component address is only the IP as kubernetes knows the port to use so the port is not scraped. I changed it to address still instead of podIP